packages/block-library/src/columns/editor.scss

```diff
@@ -148,26 +148,30 @@ div.block-core-columns.is-vertically-aligned-bottom {
 	right: 0;
 }
 
+// The empty state of a columns block has the default appenders.
+// Since those appenders are not blocks, the parent, actual block, appears "hovered" when hovering the appenders.
+// Because the column shouldn't be hovered as part of this temporary passthrough, we unset the hover style.
+.wp-block-columns [data-type="core/column"].is-hovered {
+	> .block-editor-block-list__block-edit::before {
+		content: none;
+	}
+
+	.block-editor-block-list__breadcrumb {
+		display: none;
+	}
+}
+
 // In absence of making the individual columns resizable, we prevent them from being clickable.
 // This makes them less fiddly. @todo: This should be revisited as the interface is refined.
 .wp-block-columns [data-type="core/column"] {
 	pointer-events: none;
 
-	// The empty state of a columns block has the default appenders.
-	// Since those appenders are not blocks, the parent, actual block, appears "hovered" when hovering the appenders.
-	// Because the column shouldn't be hovered as part of this temporary passthrough, we unset the hover style.
-	&.is-hovered {
-		> .block-editor-block-list__block-edit::before {
-			content: none;
-		}
-
-		.block-editor-block-list__breadcrumb {
-			display: none;
-		}
+	// This selector re-enables clicking on any child of a column block.
+	.block-core-columns .block-editor-block-list__layout {
+		pointer-events: all;
 	}
 }
 
-// This selector re-enables clicking on any child of a column block.
 :not(.components-disabled) > .wp-block-columns > .editor-inner-blocks > .editor-block-list__layout > [data-type="core/column"] > .editor-block-list__block-edit > * {
-	pointer-events: all;
+	//pointer-events: all;
```

Comment:
Should this whole section be removed as it doesn't contain any active CSS rules?

Reply:
Aargh, is it monday? Sorry about that, and thanks: https://github.com/WordPress/gutenberg/pull/14876/commits/118fbfe3aea9af532a6ff6677f095ca8178ae22a